ptx/S_12_6_Curl.ptx

```diff
@@ -1,106 +1,469 @@
+  </p>
+  <p>
+    Let's start by measuring the circulation around a square with side length <m>2h</m> centered at a point <m>(a,b)</m>. Namely, we will look at the line integral of our vector field as we move along the square curve shown in <xref ref="Curl_dev_square">Figure</xref>.
+  </p>
+  <figure xml:id="Curl_dev_square">
+    <caption>A square centered at the point <m>(a,b)</m> with side lengths <m>2h</m> traveled counterclockwise</caption>
+    <image width ="55%" source="images/Square_with_arrows"/>
+  </figure>
+  <p>
+    We can parameterize the top edge of the box (in blue) using the parameterization 
+      <me> \vr_{top}(t) = \langle a-t,b+h\rangle </me> 
+      where <m>-h\geq t\geq h</m>. Similarly, the bottom, right, and left can be parameterized by 
+      <me>
+        \vr_{bottom}(t) = \langle a+t,b-h\rangle \\
+        \vr_{right}(t) = \langle a+h,b+t\rangle \\
+        \vr_{left}(t) = \langle a-h,b-t\rangle
+      </me>
+  </p>
+  <p>
+    In order to measure the circulation, we want to know how much of the vector field is parallel to the direction of travel, so along each of the sides of the box, we only need to look at one of the components of the vector field. The only contribution to the circulation on the top and bottom comes from the horizontal component (<m>F_1</m>) and the vertical component (<m>F_2</m>) is all that contributes on the right and left sides. This means that we can simplify our line integrals considerably once we apply <xref ref="LineInt_Param_Thm">Theorem</xref>. For instance, the circulation on the top can be written as
+    <me>
+      \int_{top} \vF \cdot \vr=\int_{-h}^h \langle F_1,F_2\rangle \cdot \langle -1,0\rangle \enspace dt = \int_{-h}^h -F_1( a-t,b+h) \enspace dt
+    </me>
+    Similarly we can simplify the appropriate line integrals on the other sides (ordered below as top, left, bottom, right) to get the total circulation around the square to be
+    <me> 
+      \verb"Total Circulation" = \int_{\verb"Square"} \vF \cdot d\vr  \\
+      = \int_{-h}^h -F_1(a-t,b+h) \enspace dt +  \int_{-h}^h -F_2(a-h,b-t) \enspace dt \\ 
+      \quad \quad + \int_{-h}^h F_1(a+t,b-h) \enspace dt + \int_{-h}^h F_2(a+h,b+t) \enspace dt
+    </me>
+    Note that the sign on the top and the left sides end up being switched because the direction of travel on these sides is in the negative coordinate direction. It is important to note that by convention, we talk about "positive" rotation being counterclockwise and "negative" rotation being clockwise when looking at these planar graphs.
```

Comment:
Make sure to use `<q>` for things that should be in (double) quotes, as otherwise the LaTeX quotes will come out wrong.